tools/tidb-binlog-kafka.md

```diff
@@ -93,6 +93,20 @@ cd tidb-binlog-latest-linux-amd64
     dir = "/path/pb-dir"
     ```
 
+- The drainer outputs `kafka` and you need to set the following parameters in the configuration file:
+
+    ```
+    [syncer]
+    db-type = "kafka"
+
+    # when db-type is kafka, you can uncomment this to config the down stream kafka, or it will be the same kafka addrs where drainer pull binlog from.
+    # [syncer.to]
+    # kafka-addrs = "127.0.0.1:9092"
+    # kafka-version = "0.8.2.0"
+    ```
+
+    The data which outputs to kafka follows the binlog format sorted by ts and defined by protobuf. Please refer to [driver](https://github.com/pingcap/tidb-tools/tree/master/tidb_binlog/driver) to access the data and sync to the down stream.
```

Comment:
- Please refer to -> See

Note: Generally, we do not use "please" in the documentation, because it's over-polite. Reference: https://developers.google.cn/style/tone#politeness